src/Router.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App;
+
+use App\Controllers\DashboardController;
+use App\Controllers\UserManagementController;
+
+class Router
+{
+    public function route(): void
+    {
+        if (isset($_GET['controller']) && isset($_GET['do'])) {
+            $controller = htmlspecialchars($_GET['controller']);
+            $action = htmlspecialchars($_GET['do']);
+
+            switch ($controller) {
+                case 'dashboard':
+                    if ($action === 'show') {
+                        DashboardController::show();
+                    }
+                    break;
+
+                case 'usermanagement':
+                    if ($action === 'show') {
+                        UserManagementController::show(1);
+                    }
+                    break;
+
+                default:
+                    echo "Nieznany kontroler";
+                    break;
+            }
+        } else {
+            echo "Brak parametrów w URL";
```

Comment:
Jeżeli jest brak parametrów w adresie URL to nie wyrzucajmy żadnego błędu, zamiast tego zrób tutaj nową metodę np. redirectToDefaultRoute a w niej przekieruj na domyślny routing twojej aplikacji. Zalecam przekierowanie na dashboard